site/documentation/content/user-guide/mqtt-adapter.md

```diff
@@ -608,6 +608,9 @@ that gateway has created such a command subscription, that gateway will be chose
 on behalf of the command target device and that has an open subscription will be chosen randomly to receive the command
 message.
 
+It is not possible to use both topic names `command//+/req/#` and `command//[<gateway-id>]/req/#` at the same time. As
+at that case only the last such subscription would be handled from the Command Router.
+
```

Comment:
I would suggest replacing `As at that case` with `In that case` here.

Reply:
done